_config.yml

```
@@ -0,0 +1,44 @@
+# Site settings
+title: AI Research Documentation
+description: A curated collection of AI tools, frameworks, and resources
+baseurl: "/brain" # the subpath of your site, e.g. /blog
+url: "https://wizardsupreme.github.io" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)